src/others/components/Map.tsx

```diff
@@ -15,6 +54,12 @@ export function Map() {
   const [lat] = useState(ukraineCenterCoords.lat);
   const [zoom] = useState(4);
 
+  const geojson: FeatureCollection<Geometry, GeoJsonProperties> = {
+    type: "FeatureCollection",
+    //@ts-ignore
```

Comment:
*non blocking*
TypeScript is your friend 😉 See `src/others/fixtures/request.fixture.tsx`
```suggestion
```